tests/resumable_media/unit/requests/test_download.py

```diff
@@ -51,6 +51,25 @@ def test__write_to_stream_no_hash_check(self):
             chunk_size=_request_helpers._SINGLE_GET_CHUNK_SIZE, decode_unicode=False
         )
 
+    def test__write_to_stream_empty_chunks(self):
```

Comment:
Curious why this addition became necessary?

Reply:
If I recall correctly, I wrote it while trying to debug something, and kept it afterwards for system test coverage purposes. What do you think, should I keep it or remove it and maybe add it in a future PR for cleanliness reasons?

Reply:
Yeah, you're right. Thanks for bringing it up